spec/unit/library_spec.rb

```diff
@@ -113,7 +113,7 @@ def generate_software(name, version, dependencies = [])
         end
 
         it "returns an array of software descriptions, with all top level deps first, assuming they are not themselves transitive deps" do
-          expect(library.build_order).to eql(
+          expect(library.build_order(&:name)).to eql(
```

Comment:
@deriamis should that be `build_order.map(&:name)` [again](https://github.com/chef/omnibus/pull/1147/files#diff-97f442dbeade17b123a0478ae9e6a7f7d482ac4e1399ffaf82891a7ec0523438R85), or am I missing something?

Reply:
No, you aren't missing anything. I am 🤡.

This is something I was moving over from something else I was doing, and I couldn't just cherry-pick it because it was part of something larger. Looks like I missed a couple of details. I'll push up a fix.